Common/Config/Settings.Designer.cs

```diff
@@ -320,5 +313,17 @@ public sealed partial class Settings : global::System.Configuration.ApplicationS
                 this["IsInstalled"] = value;
             }
         }
+        
+        [global::System.Configuration.UserScopedSettingAttribute()]
+        [global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
+        [global::System.Configuration.DefaultSettingValueAttribute("False")]
+        public bool IsPortable {
+            get {
+                return ((bool)(this["IsPortable"]));
+            }
+            set {
+                this["IsPortable"] = value;
+            }
+        }
     }
```

Comment:
Good to have 👍

Reply:
Thanks 😄 
I guess we can also use a "log" subfolder in the application folder when portable mode is enabled (to ensure everything is indeed in the same place). This will be for a future branch...